### DISCUSSION TITLE
Are you aware of alternatives in other languages?

### DISCUSSION THREAD

**Discussion by sneko** [timgit/pg-boss#415] [General]
Hi @timgit,

Huge thanks for your work. I'm using for quite some time your library and it's now just a nobrainer when thinking about async processing.

Currently I'm moving over a PHP project and after some research I didn't find an equivalent. I just found https://github.com/que-rb/que for Ruby.

If you or anyone else have a thought about other libraries in other languages, I think it could help to share them 🙏  ! (maybe if there are enough with equivalent features the list could be referenced in the `README.md`?)

Thank you,

EDIT: maybe for PHP https://symfony.com/doc/current/messenger.html is something to look at (didn't try yet)

**Comment by timgit** [comment]
Thanks!  I don't maintain a list of other packages using SKIP LOCKED in postgres, but there are quite a few out there now.
